src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -1063,21 +1061,16 @@ ADIOS2IOHandlerImpl::adios2AccessMode( std::string const & fullPath )
         if( auxiliary::directory_exists( fullPath ) ||
             auxiliary::file_exists( fullPath ) )
         {
-            std::cerr << "ADIOS2 does currently not yet implement ReadWrite "
-                         "(Append) mode. "
-                      << "Replacing with Read mode." << std::endl;
             return adios2::Mode::Read;
```

Comment:
Now that the APPEND mode  is added.  Should READ_WRITE mode be rejected for ADIOS?  Might be semantically cleaner.

Reply:
READ_WRITE is still useful for file-based iteration encoding since it allows you to read old iterations while at the same time to write new ones. This feature is not found in any other iteration encodings. For the other iteration encodings, there is no use in having READ_WRITE mode for ADIOS2, yeah.
We should however not deactivate it immediately, but allow users to transition to APPEND mode since it's a breaking change.